cmd/ddev/cmd/stop.go

```diff
@@ -21,7 +21,7 @@ var LocalDevStopCmd = &cobra.Command{
 		err = app.Stop()
 		if err != nil {
 			log.Println(err)
-			util.Failed("Failed to stop containers for %s. Run `ddev list` to ensure your site exists.", app.ContainerName())
+			util.Failed("Failed to stop containers for %s. Run `ddev list` to ensure your site exists. error=", app.ContainerName(), err)
```

Comment:
stylistic nitpick: I _feel_ like we've largely shown errors like this as "error: " vs. "error=". IMO colon/space reads better.